.github/workflows/htmlhint_linter.yml

```diff
@@ -0,0 +1,21 @@
+name: HTML Linter
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  lint:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      
+    - name: Linter
+      # https://github.com/htmlhint/HTMLHint
+      run: npx htmlhint -r "tagname-lowercase,attr-lowercase,attr-value-double-quotes,doctype-first,doctype-html5,tag-pair,spec-char-escape,id-unique,src-not-empty,attr-no-duplication,title-require,attr-unsafe-chars,href-abs-or-rel,html-lang-require,id-unique,input-requires-label,tag-pair,tagname-lowercase,tagname-specialchars,tags-check,attr-no-unnecessary-whitespace,empty-tag-not-self-closed"
```

Comment:
I don't think we should have duplication between the list of rules in the GitHub Action and the Makefile. 2 possible solutions:

1. Run `make lint` instead of `npx ...` in the GitHub Action
2. [Create a config file](https://htmlhint.com/docs/user-guide/configuration)